src/components/VotingDialog/VotingDialog.scss

```diff
@@ -106,6 +106,17 @@
   width: 80%;
   background: $color-planning-pink;
   cursor: pointer;
+  transition: all 0.08s ease-out;
+
+  &:hover {
+    filter: brightness(1.05);
+    transform: scale(1.01);
+    box-shadow: 0 6px 9px 0 rgba(var(--accent-color-rgb), 0.24);
```

Comment:
At least make it consistent to the other shadows and use 4 8 or 8 16. Or is there a reason to use these specific values?